app/(afterLogin)/(deal)/deals/manage-deals/_components/filter/index.tsx

```
@@ -0,0 +1,93 @@
+import { Button, Form, Input, Modal, Select } from 'antd';
```

ESLint

[error] 'Modal' is defined but never used.

(@typescript-eslint/no-unused-vars)